src/platform/elements/form/Control.ts

```diff
@@ -16,42 +16,42 @@ import { DateFormatService } from '../../services/date-format/DateFormat';
 import { FieldInteractionApi } from './FieldInteractionApi';
 
 export interface IMaskOptions {
-    mask: any;
-    keepCharPositions: boolean;
-    guide: boolean;
+  mask: any;
+  keepCharPositions: boolean;
+  guide: boolean;
 };
 
 @Directive({
-    selector: 'textarea[autosize]'
+  selector: 'textarea[autosize]'
 })
 export class NovoAutoSize implements AfterContentInit {
-    @HostListener('input', ['$event.target'])
-    onInput(textArea: HTMLTextAreaElement): void {
-        this.adjust();
-    }
-
-    constructor(public element: ElementRef) { }
-
-    ngAfterContentInit(): void {
-        setTimeout(() => {
-            this.adjust();
```

Comment:
It looks like your commit changed the formatting to be 2 spaces per tab instead of 4. Did you intend to do that?

Reply:
yea. my editor does it automatically since .editorconfig is set to 2 spaces. @jgodi mentioned that he would update it to 4 once all of the table changes are done and reformat all the files in novo-elements in one go to use 4 spaces.

Reply:
Nevermind - looks like we changed that to be 2 spaces with the default .prettierrc file.